### ISSUE TITLE
Blank spaces break markdown table

### ISSUE THREAD

**Issue by avasconcelos114** [mattermost/mattermost-mobile#2461]
#### Summary
Blank spaces in certain places of Markdown tables cause them not to render at all

Note: There was a [very similar issue](https://mattermost.atlassian.net/browse/MM-11955) reported in the past. But in this issue blank spaces in other places will cause the same behavior to occur

#### Environment Information

- Device Name: 
- OS Version: iOS 12
- Mattermost App Version: 1.14 .0 (Build 158)
- Mattermost Server Version: 5.5.0

#### Steps to reproduce
1. In the mobile app, create the following tables:
  a. (With extra spaces at the end of each line)
```
| Header 1 | Header 2 | 
| :--: | :--: | 
| r1c1 | r1c2 | 
| r2c1 | r2c2 | 
| r3c1 | r3c2 | 
```
  b. (With an extra space before the second `:` )
```
| Header 1 | Header 2 |
| :--: | :-- : |
| r1c1 | r1c2 |
| r2c1 | r2c2 |
| r3c1 | r3c2 |
```
2. View it on Mobile app & Webapp

#### Expected behavior
Tables should render on mobile properly, as they already to on the webapp

#### Observed behavior
Table markdown appears in plaintext on the mobile app
![image](https://user-images.githubusercontent.com/29700158/49980393-9a145080-ff96-11e8-82c4-39fa6a2bf167.png)

#### Possible fixes
Changes may need to be made on the markdown parser used by the mobile app to ensure whitespace gets ignored in the right parts

**Comment by amyblais** [comment]
Thanks for the report, opened a ticket here: https://mattermost.atlassian.net/browse/MM-13516.

**Comment by amyblais** [comment]
Closing this as this has been fixed for v1.18 release,
